### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/204f2eb878aa1f284e723c6d363e3df242e5d90d

### CODE REVIEW

File: en/firefox/whatsnew/whatsnew-developer-mdnplus.ftl [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+# Main title
+developer-mdnplus-more-mdn-your-mdn = More { -brand-name-mdn }. <em>Your</em> { -brand-name-mdn }.
+developer-mdnplus-mdn-is-an-open-source = { -brand-name-mdn-web-docs } is an open-source, collaborative project documenting Web platform technologies, including CSS, HTML, JavaScript and Web APIs. We also provide an extensive set of learning resources for beginning developers and students.
+developer-mdnplus-support-mdn-and-make = Support { -brand-name-mdn } <em>and</em> make it your own.
+
+# CTA button
+developer-mdnplus-get-started = Get Started
+
+developer-mdnplus-whats-included = What’s included
```
Comment (peiying2) [inline en/firefox/whatsnew/whatsnew-developer-mdnplus.ftl:19]: Nothing wrong with the text. On the staging server, there is an underscore at the end of this string. Double check the code.

Comment (craigcook) [reply en/firefox/whatsnew/whatsnew-developer-mdnplus.ftl:19]: That's a purely visual flourish added by CSS, it's not in the text. I think it's to make it look programmer-y?

Comment (peiying2) [reply en/firefox/whatsnew/whatsnew-developer-mdnplus.ftl:19]: OK, approved then.

**Review by peiying2 — APPROVED** [review]

File: configs/pontoon.toml [hunk 2]
```diff
@@ -234,6 +227,9 @@ locales = [
         "nl",
         "sv-SE",
     ]
+[[paths]]
+    reference = "en/firefox/whatsnew/whatsnew-developer-mdnplus"
```
Comment (flodolo) [inline configs/pontoon.toml:231]: Pretty sure it's not expected for this to be missing `.ftl`.

Comment (craigcook) [reply configs/pontoon.toml:231]: Doh. Fixed in https://github.com/mozilla/bedrock/pull/11764. Sorry about that.

**Pull request merged**